src/rgw/rgw_rados.h

```diff
@@ -1800,9 +1800,8 @@ class RGWRados
 protected:
   CephContext *cct;
 
-  librados::Rados **rados;
-  atomic_t next_rados_handle;
-  uint32_t num_rados_handles;
+  std::vector<librados::Rados> rados;
+  uint32_t next_rados_handle;
```

Comment:
This is rather stylistic but, as `next_rados_handle` is used as an index, it would be nice to see `size_t` here.